components/binding-usage-controller/Jenkinsfile

```diff
@@ -5,9 +5,9 @@ def application = "binding-usage-controller"
 
 def isMaster = params.GIT_BRANCH == 'master'
 
-def dockerPushRoot = isMaster 
-    ? "${env.DOCKER_REGISTRY}"
-    : "${env.DOCKER_REGISTRY}snapshot/"
+def dockerPushRoot = isMaster
```

Comment:
if PR images end up in pr directory, why "master" images are not placed in the "master" directory? :)

Reply:
Well, it is debatable, but registry folders may not map one to one with branches. It is called develop because it contains development builds. For now only master builds target that dir, but that might change.

I will present this in the SIG and we can discuss it further if needed.